libipl/libipl.H

```diff
@@ -53,12 +56,9 @@ struct ipl_error_info {
 	//specilized constructors
 	ipl_error_info() : type(IPL_ERR_OK), private_data(NULL)  {}
 	ipl_error_info(ipl_error_type type) : type(type), private_data(NULL) {}
-
-	~ipl_error_info(){
-		// freeup the private_data allocated memory
-		if (private_data)
-			free(private_data);
-	}
+	ipl_error_info(ipl_error_type type, void* private_data) : type(type),
+	    private_data(private_data) {}
+	~ipl_error_info(){}
```

Comment:
free(private_data) ?

Reply:
we are just passing the address of the FFDC, I am not allocating any memory. As it is a library call we can do this if it is inter process communication may be we need to allocate memory.